.github/workflows/release.yml

```diff
@@ -53,19 +58,19 @@ jobs:
       - name: Setup Java JDK
         uses: actions/setup-java@v1.4.3
         with:
-          java-version: 11 
+          java-version: 11
 
       - name: Set Up CommandBox
-        uses: elpete/setup-commandbox@v1.0.0
-      
+        uses: Ortus-Solutions/setup-commandbox@main
+
       - name: Install and Configure Semantic Release
         run: |
           box install commandbox-semantic-release
```

Comment:
While we're changing the build script, can we pin to a major version range? `^2.0.0`?